components/centos/centos-mesos-marathon-singlenode-setup/Vagrantfile

```diff
@@ -0,0 +1,26 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+Vagrant.configure("2") do |config|
+	config.vm.box = "centos/7"
```

Comment:
I thought this would be an issue. Pretty obvious thing from your point of view, I agree. Can you help me understand if k8s is installed by default in ADB? I ask this because k8s and Marathon have a conflict of ports. Hence, I might have to modify Vagrantfile to disable k8s services.